scripts/sct_detect_pmj.py

```diff
@@ -40,9 +41,9 @@ def get_parser():
                         example="t2.nii.gz")
     parser.add_option(name="-c",
                         type_value="multiple_choice",
-                        description="type of image contrast, if your contrast is not in the available options (t1, t2), use t1 (cord bright / CSF dark) or t2 (cord dark / CSF bright)",
-                        mandatory=True,
-                        example=["t1", "t2"])
+                        description="type of image contrast, automatically detected but can be specified manually (t2: cord dark / CSF bright ; t1: cord bright / CSF dark)",
+                        mandatory=False,
+                        example=["t1", "t2", "t2s"])
```

Comment:
We do not have a model for detecting the pmj on T2star images (which is almost impossible on these images and probably never requested by the user). I would suggest to remove `t2s`. But I am maybe missing something :)

Reply:
> I would suggest to remove t2s.

agreed

Reply:
I agree with "I let the network output t1, t2 or t2s and then I map t2s to t2 since there are only two available contrasts." (quotation from the discussion). But, please correct me if I am missing something: we are here talking about the parser, we do not want the user to input `t2s` since we do not have a model for these images. Moreover, if the user specifies the argument `-c`, then the network will be bypassed.

Reply:
Exact I will delete the option t2s !

Reply:
good point!